src/main/resources/project/build.gradle.template

```diff
@@ -58,16 +59,16 @@ test {
 
 compileKotlin {
     kotlinOptions.jvmTarget = "1.8"
-    dependsOn 'generateContractWrappers'
 }
 
 compileTestKotlin {
     kotlinOptions.jvmTarget = "1.8"
-    dependsOn 'generateTestContractWrappers'
 }
 
 sourceSets {
     main.kotlin.srcDirs += "${web3j.generatedFilesBaseDir}/main/java"
```

Comment:
I think this can be removed as they are the default values for the gradle plugin